include/pika_conf.h

```diff
@@ -294,21 +294,21 @@ class PikaConf : public slash::BaseConf {
   int port_;
   std::string slaveof_;
   int slave_priority_;
-  int thread_num_;
-  int thread_pool_size_;
-  int sync_thread_num_;
+  int thread_num_ = 12;
+  int thread_pool_size_ = 12;
+  int sync_thread_num_ = 3;
   std::string log_path_;
   std::string db_path_;
   std::string db_sync_path_;
   int expire_dump_days_;
-  int db_sync_speed_;
+  int db_sync_speed_ = 1024;
   std::string compact_cron_;
   std::string compact_interval_;
-  int64_t write_buffer_size_;
-  int64_t arena_block_size_;
-  int64_t max_write_buffer_size_;
-  int max_write_buffer_num_;
-  int64_t max_client_response_size_;
+  int64_t write_buffer_size_ = 1024 * 1024 * 1024 * 10;     // 10GB
+  int64_t arena_block_size_ = 1024 * 1024 * 1024;       // 1GB
```

Comment:
default with rocksdb : 33554432   //32MB

Reply:
arena_block_size_

Reply:
i suggest use shared block cache, with big write_buffer_size

user dont know how to use rocksdb

Reply:
I agree with you that users don't know how to use rocksdb or maybe not have to. arena_block_size_ is the size that rocksdb pre malloc memory size.  it is suggested 1/8 memtable size .

Reply:
if we changed write_buffer_size size ,have we to change sst file size so that memtable size  10 times bigger than sst file size ? or maybe we have to give a benchmark before change it ?

Reply:
i'll do the benchmark